dbt/scripts/export_qc_town_close_reports.py

```diff
@@ -181,14 +181,17 @@ def main():
 
             iasworld_deps[table_name] = formatted_dep
 
-        print("Run the following commands on the Data Team server:")
+        print(
+            "Run the following commands on the Data Team",
+            "server as the shiny-server user:",
+        )
         print()
         print("cd /home/shiny-server/services/service-spark-iasworld")
-        print("docker compose up -d")
+        print("docker compose --profile prod up -d")
         print(
-            "docker exec spark-node-master ./submit.sh "
```

Comment:
Maybe a preference thing, but trailing spaces aren't needed if the args are comma-separated. But I'm fine either way.

Reply:
**[Praise]** I didn't know that, TIL!